admin/tests/integration/components/organizations/information-section-view_test.js

```diff
@@ -243,9 +243,27 @@ module('Integration | Component | organizations/information-section-view', funct
         assert.dom(screen.getByText('Organisation parente')).exists();
       });
     });
+    module('when organization is child', function () {
+      test('it should display child label', async function (assert) {
+        //given
+        const store = this.owner.lookup('service:store');
+        const parentOrganization = store.createRecord('organization', { id: 5, type: 'SCO', isManagingStudents: true });
+        const organization = store.createRecord('organization', {
+          type: 'SCO',
+          isManagingStudents: true,
+          parentOrganizationId: parentOrganization.id,
+        });
+        this.set('organization', organization);
+        // when
+        const screen = await render(hbs`<Organizations::InformationSectionView @organization={{this.organization}} />`);
+        // then
+
+        assert.dom(screen.getByText('Organisation enfant')).exists();
+      });
```

Comment:
Le but étant d'afficher 2 éléments

- le tag pour dire que c'est une organisation _fille_
- le nom de l'organisation mère

On devrait modifier le test

```suggestion
      test('displays child label and parent organization name', async function (assert) {
        //given
        const store = this.owner.lookup('service:store');
        const parentOrganization = store.createRecord('organization', { id: 5, type: 'SCO', isManagingStudents: true });
        const organization = store.createRecord('organization', {
          type: 'SCO',
          isManagingStudents: true,
          parentOrganizationId: parentOrganization.id,
          parentOrganizationName: 'Shibusen',
        });
        this.set('organization', organization);
        
        // when
        const screen = await render(hbs`<Organizations::InformationSectionView @organization={{this.organization}} />`);
        
        // then

        assert.dom(screen.getByText('Organisation enfant')).exists();
        assert.dom(screen.getByText('Shibusen')).exists();
      });
```